src/index.tsx

```diff
@@ -18,6 +19,7 @@ async function renderApp() {
     environment: process.env.NODE_ENV
   });
 
+  await initializeArc();
```

Comment:
is this a good idea? If this fails or takes a long time, we have no good way of showing user feedback...